DonoGear/app/src/main/java/com/example/donogear/actionpages/MainActivity.java

```diff
@@ -97,8 +112,14 @@ private void getFilters() {
 
     /**
      * Queries all the collectible-data from the database
```

Comment:
This method javadoc is confusing. If it queries all of these things, just make it one sentence and combine all 3 statements